phpBB/phpbb/di/extension/core.php

```diff
@@ -26,7 +26,11 @@
 */
 class core extends Extension
 {
-	const TWIG_OPTIONS_POSITION = 7;
+	/**
```

Comment:
You will have to remove this again, because this have 7 parameters again after your changes

Reply:
This should be correct again since I added back the DI of assets bag